paper/paper.md

```diff
@@ -63,11 +63,11 @@ Platforms such as MassBank [@horai_massbank_2010], GNPS [@Wang2016] or MetaboLig
 Without suitable tools, however, exploitation of such datasets remains overly challenging. 
 In particular, large collected datasets contain data acquired using different instruments and measurement conditions, and can further contain a significant fraction of inconsistent, wrongly labeled, or incorrect metadata (annotations).
 
-``matchms`` is an open-access Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
+``matchms`` is an open-source Python package to import, process, clean, and compare mass spectrometry data (MS/MS) (see \autoref{fig:flowchart}).
 It allows to implement and run an easy-to-follow, easy-to-reproduce workflow from raw mass spectra to pre- and post-processed spectral data. 
-Raw data can be imported from the commonly used formats msp, mzML [@martens_mzmlcommunity_2011], mzXML, MGF (mzML, mzXML, MGF file importers are built on top of pyteomics [@levitsky_pyteomics_2019][@goloborodko_pyteomicspython_2013]), as well as from json files (as provided by GNPS), but also via Universal Spectrum Identifiers (USI) [@wang_interactive_2020]. Further data formats or more extensive options regarding metadata parsing can best be handled by using pyteomics [@levitsky_pyteomics_2019] or pymzml [@kosters_pymzml_2018].
+Raw data can be imported from the commonly used formats msp, mzML [@martens_mzmlcommunity_2011], mzXML, MGF (mzML, mzXML, MGF file importers are built on top of pyteomics [@levitsky_pyteomics_2019;@goloborodko_pyteomicspython_2013], as well as from JSON files (as provided by GNPS), but also via Universal Spectrum Identifiers (USI) [@wang_interactive_2020]. Further data formats or more extensive options regarding metadata parsing can best be handled by using pyteomics [@levitsky_pyteomics_2019] or pymzml [@kosters_pymzml_2018].
```

Comment:
This is a minor formatting fix for the citation rendering.

Reply:
Perfect, thanks.